securedrop/source_templates/index.html

```diff
@@ -89,11 +89,11 @@ <h2 class="welcome-text">
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. #}
     <div class="bubble" id="security-slider-info-bubble">
-      <p>{{ gettext('You appear to be using the Tor Browser. You can turn the Security Slider to Safest in 4 easy steps!') }}</p>
+      <p>{{ gettext('You appear to be using the Tor Browser. You can move the Security Slider to Safest in 4 easy steps!') }}</p>
```

Comment:
"You can adjust the security slider to "Safest", by doing these 4 things"
(avoids explaining it as "easy", which is redundant. Possibly "Adjust the security slider to "Safest" by" suffices)